imap_processing/tests/codice/test_codice_l2.py

```diff
@@ -440,7 +440,7 @@ def test_codice_l2_sw_angular_intensity(mock_get_file_paths, codice_lut_path):
         / "codice"
         / "data"
         / "l2_validation"
-        / "imap_codice_l2_lo-sw-angular_20250814_v007.cdf"
+        / "imap_codice_l2_lo-sw-angular_20250814_v008.cdf"
```

Comment:
I feel like we've been bumping the version number a lot lately, can you put the version as a format string at the top of the file instead of updating it in so many places?

```suggestion
        / "imap_codice_l2_lo-sw-angular_{VALIDATION_FILE_DATE}_{VALIDATION_FILE_VERSION}.cdf"
```

at the top of this file then set:
```python
VALIDATION_FILE_DATE = "20250814"
VALIDATION_FILE_VERSION = "v008"
```